spec/spec_helper.rb

```diff
@@ -27,6 +27,14 @@
   # rspec-expectations config goes here. You can use an alternate
   # assertion/expectation library such as wrong or the stdlib/minitest
   # assertions if you prefer.
+  config
+    .before(:suite) do
```

Comment:
It's weird that this `.before` is indented and on a separate line. `config.before(:suite) do` reads better I think?

Reply:
@swcraig I agree, this stupid prettier ruby-plugin occasionally does weird things.